js/os_test.ts

```diff
@@ -34,9 +34,21 @@ test(async function statSyncSuccess() {
   assert(testingInfo.isDirectory());
   assert(!testingInfo.isSymlink());
 
-  const srcInfo = deno.statSync("src");
+  const srcInfo = deno.statSync("src/");
```

Comment:
Is there a functional reason for `"src/"` instead of `"src"`? Also below you use `"js/."`
... It's an odd way to write these and I'm wondering if you're doing it for extra clarity?

Reply:
@ry `src/` is just for clarity. `js/.` is to test that the name of that path is `js`.